### PR TITLE
Add ubuntu 22

### PR DESCRIPTION
**Pull request by LatorreDev**

### CODE REVIEW

File: installation/linux/ubuntu.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Ubuntu
 
-Fluent Bit is distributed as **fluent-bit** package and is available for the latest stable Ubuntu system: Focal Fossa.
+Fluent Bit is distributed as **fluent-bit** package and is available for the latest stable Ubuntu system: Jelly Jellyfish.
```
Comment (lecaros) [inline installation/linux/ubuntu.md:3]: it's Jammy Jellyfish.

File: installation/getting-started-with-fluent-bit.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ description: The following serves as a guide on how to install/deploy/upgrade Fl
...
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | CentOS / Red Hat       | [CentOS 7](linux/redhat-centos.md#install-on-redhat-centos), [CentOS 8](linux/redhat-centos.md#install-on-redhat-centos)                                                                                |
-| Ubuntu                 | [Ubuntu 16.04 LTS](linux/ubuntu.md#ubuntu-16-04-lts-xenial-xerus), [Ubuntu 18.04 LTS](linux/ubuntu.md#ubuntu-18-04-lts-bionic-beaver), [Ubuntu 20.04 LTS](linux/ubuntu.md#ubuntu-20-04-lts-focal-fossa) |
+| Ubuntu                 | [Ubuntu 16.04 LTS](linux/ubuntu.md#ubuntu-16-04-lts-xenial-xerus), [Ubuntu 18.04 LTS](linux/ubuntu.md#ubuntu-18-04-lts-bionic-beaver), [Ubuntu 20.04 LTS](linux/ubuntu.md#ubuntu-20-04-lts-focal-fossa),[Ubuntu 22.04 LTS](linux/ubuntu.md#ubuntu-22-04-lts-jammy-jellyfish) |
```
Comment (lecaros) [inline installation/getting-started-with-fluent-bit.md:20]: Add space after the comma.
Check the link. I don't think it's taking you anywhere on the page (ubuntu.md#ubuntu...)

File: pipeline/inputs/windows-event-log-winevtlog.md [hunk 3]
```diff
@@ -13,9 +13,9 @@ The plugin supports the following configuration parameters:
 | Interval\_NSec | Set the polling interval for each channel (sub seconds. \(optional\) | 0 |
 | Read\_Existing\_Events | Whether to read existing events from head or tailing events at last on subscribing. \(optional\) | False |
 | DB | Set the path to save the read offsets. \(optional\) |  |
-| String\_Inserts | Whether to include StringInserts in output records. \(optional\) | False  |
+| String\_Inserts | Whether to include StringInserts in output records. \(optional\) | True  |
```
Comment (lecaros) [inline pipeline/inputs/windows-event-log-winevtlog.md:16]: I think your head from your branch 1.9 was behind the latest commit (https://github.com/fluent/fluent-bit-docs/commit/bbecdc60291aa454724d80e672a31ecfead34454)
Make sure you always pull all the changes before branching.

**Review by lecaros — CHANGES_REQUESTED** [review]

File: installation/linux/ubuntu.md (hunk 1 above)
Comment (LatorreDev) [reply installation/linux/ubuntu.md:3]: I checked this using markdown live inspector
ubuntu22

File: pipeline/inputs/windows-event-log-winevtlog.md (hunk 3 above)
Comment (LatorreDev) [reply pipeline/inputs/windows-event-log-winevtlog.md:16]: Updated to lastest changes 
image

**Pull request closed without merging**